### DISCUSSION TITLE
hide my api??

### DISCUSSION THREAD

**Discussion by hjonathan2019** [coinfoundry/miningcore#1486] [General]
any way to "hide" my public api? im talking about avoid people to "see" my api 

http://api.xxxxxx:4000/api/pools/xxxxxx/ (i have this public and i dont like that way)

**Comment by MiningCryptoLive** [comment]
Yes, run the API on 127.0.0.1:4000/api

**Reply by hjonathan2019** [reply]
Mmm im afraid asking jajaja how?? Where i have to change it??

**Comment by MiningCryptoLive** [comment]
In miningcore-ui.js what frontend are you using?

**Reply by hjonathan2019** [reply]
Miningcore.WebUI-master

**Reply by hjonathan2019** [reply]
do you known another webui better or nice?? jajaja

**Comment by MiningCryptoLive** [comment]
What's your pool address and I will look at it and let you know

**Reply by hjonathan2019** [reply]
Erg-latam-Pool.com.ar

**Comment by MiningCryptoLive** [comment]
There are additions you can do.

**Reply by hjonathan2019** [reply]
???? What???

**Comment by MiningCryptoLive** [comment]
I have another UI.

**Reply by hjonathan2019** [reply]
@MiningCryptoLive email you

**Comment by MiningCryptoLive** [comment]
In config.json

"api": {
        "enabled": true,
        "ssl": true,
        "listenAddress": "127.0.0.1",
	   "sslConfig": {
        "enabled": true,
        "sslPath": "/path/to/certificate.pfx",
        "sslPassword": "certificate password*"
      },

if no ssl set to ssl false

**Reply by MiningCryptoLive** [reply]
Then in Miningcore.UI cd assets/js and edit miningcore-ui.js. at top change api to 127.0.0.1
